plugin-server/src/main/ingestion-queues/session-recording-v2/sessions/s3-session-batch-writer.ts

```diff
@@ -44,6 +46,8 @@ class S3SessionBatchFileWriter implements SessionBatchFileWriter {
                 Body: this.stream,
                 ContentType: 'application/octet-stream',
             },
+            // 1 for dev to reduce S3 throttling
+            queueSize: isDevEnv() ? 1 : undefined,
```

Comment:
setting to undefined to accept the default as before 
(it's 4 which we could set explicitly but then on some dependency update in the future we'd miss the default changed)